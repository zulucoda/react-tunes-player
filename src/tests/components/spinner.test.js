import React from 'react';
import { render } from '@testing-library/react';
import Spinner from '../../components/spinner';

describe('Spinner - Unit Test', () => {
  test('render Spinner when showSpinner is true', () => {
    const { container } = render(<Spinner showSpinner={true} />);
    expect(container.firstChild).toMatchSnapshot();
  });

  test('render null when showSpinner is false', () => {
    const { container } = render(<Spinner showSpinner={false} />);
    expect(container.firstChild).toMatchSnapshot();
  });
});
