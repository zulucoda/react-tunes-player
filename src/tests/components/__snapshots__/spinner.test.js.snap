// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Spinner - Unit Test render Spinner when showSpinner is true 1`] = `
<div
  class="sc-bdVaJa fUXyBp"
/>
`;

exports[`Spinner - Unit Test render null when showSpinner is false 1`] = `null`;
